src/api/repositories/UserRepository.ts

```diff
@@ -0,0 +1,15 @@
+import { env } from '../../env'
+
+export default class UserRepository implements UserRepositoryInterface {
+    constructor() {}
+
+    public compareCredentials (username, password) {
+        if (username != env.app.username || password != env.app.password) return undefined;
```

Comment:
Suggestion: to reduce cognitive processing time, avoid using negation when doing comparison.